Helpjuice/ckeditor5
# Change 'open link in a new tab' to be optional

## Describe the changes in this pull request

The editor had an option called [`addTargetToExternalLinks`](https://ckeditor.com/docs/ckeditor5/latest/features/link.html#adding-target-and-rel-attributes-to-external-links) set to `true`. This option makes every link that starts with the protocol open in a new tab.

I removed this option from our editor config.

### Before & After screenshots (if this pull request changes the UI)

#### Master

https://user-images.githubusercontent.com/28270050/194393675-71a765f2-a0e0-4f8d-8fe8-a71f8635c728.mp4

#### This PR

https://user-images.githubusercontent.com/28270050/194393708-71c47df2-29f8-4724-a87a-1e1b78ceed9c.mp4

### Does this resolve an existing issue?

[sc-549]

### How should this be tested during the review?

Open `/packages/ckeditor5-build-helpjuice/sample/index.html` and try to add a link. By default, it should be a regular link. Editing it and toggling ***Open Link in a New Tab*** adds the attributes for doing so. Turning it off should remove them.
